### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by XsidX**
> For this PR, I:
  - [use serial data_type for primary key](https://github.com/XsidX/vet-clinic-db/commit/f2d1de3c00fedef4ffe062a5aaa1b8a261dea376)
  - [add species column](https://github.com/XsidX/vet-clinic-db/commit/5a683c3bd836fffc6aa5d1ad82c8abbbdab6a330)
  - [insert 7 more records](https://github.com/XsidX/vet-clinic-db/commit/25e88d09e40461480a266b3fdea97267cc9fd506)
  - [add some more queries & transactions](https://github.com/XsidX/vet-clinic-db/commit/fd3e05b0cfc18cd6ff3962923e2633c6da392fd6)
  - [add screenshots for transactions and new queries](https://github.com/XsidX/vet-clinic-db/commit/b598f3bfde075ef56ff30d4a63d50732087337b5)
  - [add comments too describe the queries](https://github.com/XsidX/vet-clinic-db/pull/2/commits/da1a789a6cda37fa8eaf1d5152befe36c4c51b89)

### Screenshots

> **update species column in a transaction**
update-species

> **delete all records using a transaction**
delete-all-records

 > **roll back records deletion**
delete-all-records-after-rollback

> **delete-where-born-after-jan1st2022 using a transaction**
delete-where-born-after-jan1st2022

> **add savepoint _save_1_**
add-save-point-1

> **update weight to -ve weight**
update-weight-to-negative-weight

> **rollback to savepoint _save_1_**
rollback-to-savepoint-1

> **output from rollback to savepoint _save_1_** 
rollback-to-savepoint-output

> **update weight where -ve**
update-weight-where-negative

> **commit transactions**
commit-transactions

> **final output after commit**
output-after-commit

> **animal count**
animal-count

> **never escaped count**
never-escaped

> average animal weight
average-animal-weight

> **neutered vs neutered, whose escaped more**
neutured-vs-not-neutered-whoescapedmore

> **species min and max weight**
species-min-and-max-weight

>**species average escape attempts**
species-avg-escape-attempts

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,45 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+
+COMMIT;
+
+
+SELECT COUNT(*) AS animal_count FROM animals
+SELECT COUNT(*) AS never_escaped FROM animals WHERE escape_attempts < 1
+SELECT AVG(weight_kg) AS avg_animal_weight FROM animals
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals GROUP BY species
```
Comment (tufoinnkuo10) [inline queries.sql]: - [ ] Great job. You did a wonderful job with the queries and screenshot. Please add comments of the query questions you are answering here to ease the viewer know what you did and help you as well in the future.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @XsidX 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ Documented your work professionally
✔️ screenshots

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: queries.sql (hunk 1 above)
Comment (XsidX) [reply queries.sql]: @tufoinnkuo10 
thanks for the suggestion 👍🏽, 
but this is not a strict requirement to warrant **required changes**

**Review by brytebee — APPROVED** [review]
Hi, @XsidX👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design and queries.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



##

###  **Optional Suggestions:**
- N/A

##

Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**